### DISCUSSION TITLE
using iconify icons with unocss in a web BROWSER/plain html (no bundler)

### DISCUSSION THREAD

**Discussion by fessacchiotto** [unocss/unocss#3067] [Q&A] [answered]
hi! what should I include in the browser to use iconify icons with unocss?
I'm able to use them like this, but the unocss syntax is not working...
any suggestions? thanks in advance!
``` html 
        <script src="https://cdn.jsdelivr.net/npm/iconify-icon@1.0.8/dist/iconify-icon.min.js"></script>
            <!--- this doesn't work ----->
            <div class="i-mdi-alarm text-orange-400 text-4xl"></div>
           <!-- this works! ----->
            <div class="text-orange-400 text-4xl">
                <iconify-icon icon="mdi:home"></iconify-icon>
            </div>
```

**Comment by Shyam-Chen** [comment]
https://stackblitz.com/edit/vitejs-vite-3qhsv2?file=vite.config.ts,index.html,src%2Fmain.ts,src%2FApp.vue&terminal=dev

**Reply by fessacchiotto** [reply]
Thank you! 
I was wondering if we can have it working inside an HTML file (no frameworks at all), which is then uploaded to a remote http server/facility (such as, for example Cloudflare Pages, or similar).
In other words, a simple html file transferred to a remote web server.
```html
<html>
<head>
...
</head>
<body>
            <script src="https://cdn.jsdelivr.net/npm/@unocss/runtime/uno.global.js"></script>
            <script src="https://cdn.jsdelivr.net/npm/iconify-icon@1.0.8/dist/iconify-icon.min.js"></script>
            <!--- this doesn't work ----->
            <div class="i-mdi-alarm text-orange-400 text-4xl"></div>
           <!-- this works! ----->
            <div class="text-orange-400 text-4xl">
                <iconify-icon icon="mdi:home"></iconify-icon>
            </div>
</body>
</html>
```

**Comment by zyyv** [comment, marked as answer]
Accord to your code, i think you are using `runtime` package, sorry for uno not provide `presetIcons` package, so not support use icons.

**Reply by fessacchiotto** [reply]
Thank you!
